karaf/4.1.2/Dockerfile

```diff
@@ -50,7 +44,7 @@ RUN export KARAF_DIST=apache-karaf-${KARAF_VERSION}.tar.gz
     bin/client -r 10 -d 5 system:version && \
     bin/stop && \
     rm -rf instances/* && \
-    sed -e "s/osgi:/stdout, osgi:/" -i etc/org.ops4j.pax.logging.cfg
+    sed -e "s/OFF/TRACE/" -i etc/org.ops4j.pax.logging.cfg
```

Comment:
Is `TRACE` what is wanted here?

Reply:
Yes.  This is symptomatic of a breaking logging configuration change in Karaf 4.1.2.  I will need to look up the details to give a more precise explanation, but this affects a filter for messages that go to the console appender;  It provides an _additional_ filter to the messages that come from the root logger.  If root is `DEBUG` but the filter on the console appender is `INFO`, the message will be ignored.  Likewise, if the filter is `TRACE`, then it will log all messages that come from the root logger.  That's what this is doing.  It's completely different from Karaf 4.0.9.